lib/process_sonos_command.js

```diff
@@ -29,9 +29,12 @@ export default async function process_sonos_command(received_text) {
   } else if (received_text_lower.startsWith('spotify:')) {
     service_type = 'spotify';
     sonos_instruction = 'spotify/now/' + received_text;
-  } else if (received_text_lower.startsWith('tunein:')) {
+  } else if (received_text_lower.startsWith('tunein/')) {
```

Comment:
```suggestion
  } else if (received_text_lower.startsWith('tunein:')) {
```

For consistency with other URIs (e.g. Spotify, Apple, let's accept using a colon as delimiter. If you want to ALSO accept a slash, that's fine too. But for consistency NFC card will have a string: `tunein:{station ID}`.